File: Python/44.py
import os
n1=0
n2=0
n3=0
n4=0
n5=0
somapara=0
somacu=0
somacili=0
somatudo=0
somapir=0
somacon=0
volpara= 0
volcu=0
volcili=0
volpir=0
volcon=0
i=1
aresta=0
a=0
b=0
c=0
raio=0
altc=0
raio1=0
alt1=0
alt2=0
raio2=0
print("\n\nEsse algoritmo calcula o volume total de armazenamento reservatorio.Com n tanques de formatos diferentes !")
n1=int(input("\n\nDigite o numero e tanques de formato cubico: "))
for i in range(1,n1+1):
    i=i+1
    aresta=float(input("Digite o valor da aresta : "))
    volcu = (aresta*aresta)*aresta
    somacu = somacu +volcu
    print(f"\n\nEsse e o volume  do(s) reservatorio(s) de formato cubico: {somacu}")
n2=int(input("\n\nDigite o numero e tanques de formato de paralelepipedo: "))
for i in range (1,n2+1):
    i=i+1
    a=float(input("\n\nDigite a altura: "))
    b=float(input("\n\nDigite o comprimento: "))
    c=float(input("\n\nDigite a largura: "))
    volpara= (a*b)*c
    somapara = somapara +volpara
    print(f"\n\nEsse e o volume  do(s) reservatorio(s) de formato paralelepipedo: {somapara}")
n3=int(input("\n\nDigite o numero e tanques de formato de cilindro: "))
for i in range (1,n3+1):
    i=i+1
    alt=float(input("\n\nDigite a altura: "))
    raio=float(input("\n\nDigite o raio: "))
    volcili= (alt*3.14)*raio**2
    somacili = somacili +volcili
    print(f"\n\nEsse e o volume  do(s) reservatorio(s) de formato cilindrico: {somacili}")
i=1
n4=int(input("\n\nDigite o numero de tanques no formato piramidal: "))
while(i<=n4):
    raio1=float(input("\n\nDigite o valor do raio : "))
    alt1=float(input("\n\nDigite o valor da altura : "))
    volpir=(raio1*alt1)/3
    somapir=somapir+volpir
    i=i+1
    print(f"\n\nEsse e o volume  do(s) reservatorio(s) de formato piramidal: {somapir}")
i=1
n5=int(input("\n\nDigite o numero de tanques no formato conico: "))
while(i<=n5):
    raio2=float(input("\n\nDigite o valor do raio : "))
    alt2=float(input("\n\nDigite o valor da altura : "))
    volcon=(raio2*alt2)*3.14
    somacon=somacon+volcon
    i=i+1
    print(f"\n\nEsse e o volume  do(s) reservatorio(s) de formato conico: {somacon}")  
somatudo= (somacu + somapara)+somacili+somapir+somacon
print(f"\n\nEsse é o valor total do reservatorio : {somatudo}")
    
print("\n\nObrigado por ultilizar  nosso programa")

    
    
    
    
    
    


os.system("pause")

File: C/44.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float comp,alt,raio,totalGeral,totalCilindros,totalCubos,totalParal,totalCone,totalPiram;
	int op,i,n,i2,n2;
	totalGeral=0;
	totalCilindros=0;
	totalCubos=0;
	totalParal=0;
	totalCone=0;
	totalPiram=0;
 //cilíndricos, cúbicos, de  paralelepípedos,  cônicos  e  de  pirâmides.
	printf("Esse programa determina o volume total comportado por N tanques com formatos semelhantes e o volume total!\n\n");
		
	printf("Digite a quantidade total de tanques: ");
	scanf("%d", &n2);
	for(i2=1;i2<=n2;i2++){
		printf("\n\nMenu\n1- cilíndro\n2- cúbo\n3- paralelepípedo\n4- cône\n5- pirâmide");
		printf("\n\nEscolha o tipo de tanque: ");
		scanf("%d", &op);
	
		switch(op){
			case 1:
				printf("Digite a quantidade de cilíndros: ");
				scanf("%d", &n);
				i2 = i2+n;
				for(i=1;i<=n;i++){
					raio=0;
					alt=0;
					printf("Digite a altura do cilíndro %d: ",i);
					scanf("%f", &alt);
					printf("Digite o raio do cilíndro %d: ",i);
					scanf("%f", &raio);		
					totalCilindros = totalCilindros+(3.14*(raio*raio)*alt);
				}
				break;
			case 2:
				printf("Digite a quantidade de Cúbos: ");
				scanf("%d", &n);
				i2 = i2+n;
				for(i=1;i<=n;i++){
					alt=0;
					printf("Digite um dos lados do cúbo %d: ",i);
					scanf("%f", &alt);
					totalCubos = totalCubos+(alt*alt*alt);
				}
				break;
			case 3:
				printf("Digite a quantidade de paralelepípedo: ");
				scanf("%d", &n);
				i2 = i2+n;
				i=1;
				while(i<=n){
					comp=0;
					raio=0;
					alt=0;
					printf("Digite a comprimento do paralelepípedo %d: ",i);
					scanf("%f", &comp);
					printf("Digite a largura do paralelepípedo %d: ",i);
					scanf("%f", &raio);	
					printf("Digite a altura do paralelepípedo %d: ",i);
					scanf("%f", &alt);		
					totalParal = totalParal+(alt*raio*comp);
					i++;
				}
				break;
			case 4:
				printf("Digite a quantidade de cônes: ");
				scanf("%d", &n);
				i2 = i2+n;
				for(i=1;i<=n;i++){
					raio=0;
					alt=0;
					printf("Digite a raio do cône %d: ",i);
					scanf("%f", &raio);
					printf("Digite a altura do cône %d: ",i);
					scanf("%f", &alt);			
					totalCone = (((raio*alt)*3.14)+totalCone);
				}
				break;
			case 5:
				printf("Digite a quantidade de pirâmides: ");
				scanf("%d", &n);
				i2 = i2+n;
				i=1;
				do{
					comp=0;
					alt=0;
					printf("Digite a area da pirâmide %d: ",i);
					scanf("%f", &comp);
					printf("Digite a altura da pirâmide %d: ",i);
					scanf("%f", &alt);		
					totalPiram = totalPiram+((alt*raio)/3);
					i++;
				}while(i<=n);
				break;
			default:
				printf("Opção invalida!");
		}
	}
	totalGeral = totalCilindros+totalCubos+totalParal+totalCone+totalPiram;
	system("cls");
	
	printf ("\n\n\tRESULTADOS");	
	printf ("\n\nO volume total dos cilindros é de %.2f", totalCilindros);
	printf ("\n\nO volume total dos cubos é de %.2f", totalCubos);
	printf ("\n\nO volume total dos paralelepipedos é de %.2f", totalParal);
	printf ("\n\nO volume total dos cônes é de %.2f", totalCone);
	printf ("\n\nO volume total das piramides é de %.2f", totalPiram);
	printf ("\n\n\tO Valor total é de %.1f!",totalGeral);
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}


